src/js/internal/fuzzilli-reprl.ts

```
@@ -0,0 +1,326 @@
+    const n = fs.readSync(REPRL_DRFD, script_data, total_read, script_size - total_read, null);
+    if (n === 0) break;
+    total_read += n;
+  }
+
+  const script = script_data.toString("utf8");
+
+  // Execute script
+  let exit_code = 0;
+  try {
+    // Use indirect eval to execute in global scope
+    (0, eval)(script);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)